e2e/terraform/provision-nomad/main.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   provision_script = var.platform == "windows_amd64" ? "C:/opt/provision.ps1" : "/opt/provision.sh"
 
-  custom_path = abspath("${var.config_path}/custom/")
+  custom_path = dirname("${path.root}/config/custom/")
```

Comment:
should var.config_path still be used?  otherwise I can remove

Reply:
It looks like that was the only consumer. Let's get rid of it!